sdk/ml/azure-ai-ml/azure/ai/ml/constants/_job/job.py

```diff
@@ -85,6 +85,7 @@ class EntityNames:
         SSH = "ssh"
         TENSOR_BOARD = "tensor_board"
         VS_CODE = "vs_code"
+        RAY_DASHBOARD = "RayDashboard"
```

Comment:
should this be underscore_case instead, to align with the rest?

Reply:
@rtanase My understanding is
`jupyter_lab`, `ssh`, `tensor_board`, `vs_code` are interactive services exposed to customer. it's aligning with python naming syntax.

![image](https://user-images.githubusercontent.com/1989690/235990908-a38dd2c9-ac3f-40ed-a827-7cf8fb634700.png)

`Studio`, `Tracking`, `RayDashboard` are services added from service side. it's aligning with service naming syntax.
And also it might prevent someone accidentally expose this service type to customer.


![image](https://user-images.githubusercontent.com/1989690/235991076-daaa0173-b11e-4ec9-a014-ea1e07942e3f.png)